client/src/components/Checkout/Checkout.js

```diff
@@ -1,264 +1,196 @@
+                                            name="postal code"
+                                            type="text"
+                                            label="Postal code"
+                                            placeholder="Enter postal code"
+                                        />
+                                    </div>
+                                </div>
+                            )}
+                        </div>
 
-            <div className="card">
-              <div className="dropdown-header" onClick={toggle1}>
-                <h4 className="dropdown-text">Delivery option</h4>
-                <span className="dropdown-selection">
-                  {selected1 ? "-" : "+"}
-                </span>
-              </div>
-              {selected1 && (
-                <div className="wrapper">
-                  <div className="row">
-                    <div className="col-md-6">
-                      <div className="info-group mb-3">
-                        <label>First name</label>
-                        <input
-                          type="text"
-                          className="info-input"
-                          name="firstname"
-                        />
-                      </div>
-                    </div>
-                    <div className="col-md-6">
-                      <div className="info-group mb-3">
-                        <label>Last name</label>
-                        <input
-                          type="text"
-                          className="info-input"
-                          name="firstname"
-                        />
-                      </div>
-                    </div>
-                    <div className="col-md-6">
-                      <div className="info-group mb-3">
-                        <label>Phone number</label>
-                        <input
-                          type="text"
-                          className="info-input"
-                          name="firstname"
-                        />
-                      </div>
+                        <div className="card">
+                            <div className="dropdown-header" onClick={toggle1}>
+                                <h4 className="dropdown-text">Delivery option</h4>
+                                <span className="dropdown-selection">
+                                    {selected1 ? "-" : "+"}
+                                </span>
+                            </div>
+                            {selected1 && (
+                                <div className="wrapper">
+                                    <div className="row">
+                                        <LabeledInput
+                                            type="radio"
+                                            className="col-md-2"
+                                            name="delivery"
+                                            value="standard-delivery"
```

Comment:
i think you'll need to add in a value prop if you decide to use this with your LabeledInput component